src/languages/idl.js

```diff
@@ -0,0 +1,58 @@
+/*
+Language: OMG IDL
```

Comment:
Nit: regarding the name of the file, remember there are other IDLs, like web IDL. Maybe call this omgidl.js?